.github/workflows/releaseChart.yaml

```diff
@@ -1,8 +1,15 @@
 name: Release newrelic prometheus configurator chart
+# This workflow manually triggers the automated process of release based on the changelog.
+# Is recommended to run `make release-changelog` before to see an example of what is going to be generated.
```

Comment:
would make sense to create another separate target for the chart?

Reply:
There is a separate target `make release-changelog-chart`. It was a copy-paste issue